sources/console.c

```diff
@@ -1825,6 +1839,15 @@ int od_console_query(od_client_t *client, machine_msg_t *stream,
 
 	return 0;
 
+bad_role:
```

Comment:
maybe `incorrect_role`? `bad_query` is also not the best naming, we need to fix this, but not necessary in this PR